components/Checkout.php

```diff
@@ -262,6 +257,28 @@ public function onDeletePaymentProfile()
         return $result;
     }
 
+    public function onValidate()
+    {
+        $data = post();
+        $data = $this->processDeliveryAddress($data);
+
+        try {
```

Comment:
Try block might be redundant since TI handles ValidationException nicely and error messages are available on the ajax response

See https://github.com/tastyigniter/TastyIgniter/blob/master/app/main/classes/MainController.php#L397-L401

Reply:
Yep it can be removed